spring-streaming-app/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.1.6</version>
+		<version>3.2.0</version>
```

Comment:
You don't want 3.2.1?

Reply:
I thought 3.2.1 was out, but I didn't see it out in maven:

https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot/

Reply:
Hmmm... it doesn't show up in that list, but I can hit it directly:

https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot/3.2.1/

Reply:
OK, It's been updated to 3.2.1.  @trisberg

Reply:
Maybe the repo index is outdated - it is listed here https://search.maven.org/artifact/org.springframework.boot/spring-boot-starter